#! /bin/sh /usr/share/dpatch/dpatch-run
## 20Fix_desktopfile.dpatch by  <cody-somerville@ubuntu.com>
##
## DP: Removed encoding field from desktop file

@DPATCH@
diff -urNad catfish-0.3~/catfish.desktop catfish-0.3/catfish.desktop
--- catfish-0.3~/catfish.desktop	2007-07-27 18:12:52.000000000 -0700
+++ catfish-0.3/catfish.desktop	2008-02-18 16:50:06.000000000 -0800
@@ -1,6 +1,5 @@
 [Desktop Entry]
 Version=1.0
-Encoding=UTF-8
 Type=Application
 Name=Catfish
 Comment=File search
